src/main/java/core/basesyntax/service/impl/FilesWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FilesWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+
+public class FilesWriterImpl implements FilesWriter {
```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```

Reply:
I've tried this already. And it has some conflict with FileWriter in the method.